File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/identity/services/user_service.py
"""UserService for the identity domain."""

from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class UserService:
    """Application service exposing user operations."""

    def create(self, input: Optional[User]) -> User:
        """Create a new user from a partial input."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Look up a user by its identifier."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List users up to the given limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete a user by its identifier."""
        raise NotImplementedError("not implemented")

File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/identity/services/UserService.ts
import { User } from '../aggregates/User';

/** UserService service. */
export class UserService {
  create(input: Partial<User>): User { throw new Error('not implemented'); }
  findById(id: string): User | null { throw new Error('not implemented'); }
  list(limit: number): User[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}
